app/src/main/java/org/kinecosystem/kinit/viewmodel/earn/QuestionnaireViewModel.kt

```diff
@@ -95,10 +106,11 @@ open class QuestionnaireViewModel(restoreState: Boolean) :
         }
     }
 
-    fun onBackPressed() {
-        val task = taskRepository.taskInProgress
+
+    fun onCloseEvent() {
```

Comment:
this can be private method and can be renamed to `sendCloseEvent` or `reportCloseEvent`